pocs/core.py

```diff
@@ -461,6 +469,8 @@ def _check_environment(self):
             os.makedirs("{}/logs".format(pandir))
 
     def _check_messages(self, queue_type, q):
+        # TODO(jamessynge): Where do these messages come from? PAWS?
+        # TODO(jamessynge): Determine what to do about the dome.
```

Comment:
Regarding comment below, I don't think this needs to be changed.

Reply:
Do the messages come from PAWS?